theia-plugin/src/extension.ts

```diff
@@ -1,21 +1,174 @@
+// tslint:disable:no-var-requires
+// tslint:disable:no-require-imports
+// tslint:disable:no-use-before-declare
 
 /**
  * Generated using theia-plugin-generator
  */
 
 import * as theia from '@theia/plugin';
+const tooling = require('jsonforms-tooling');
 
-export function start(context: theia.PluginContext) {
-    const informationMessageTestCommand = {
-        id: 'hello-world-example-generated',
-        label: "Hello World"
-    };
-    context.subscriptions.push(theia.commands.registerCommand(informationMessageTestCommand, (...args: any[]) => {
-        theia.window.showInformationMessage('Hello World!');
-    }));
+export const start = (context: theia.PluginContext) => {
+  const createExampleProjectCommand = {
+    id: 'create-example-project',
+    label: 'JSONForms: Create Example Project',
+  };
+  const createExampleProject = theia.commands.registerCommand(
+    createExampleProjectCommand,
+    (args: any) => {
+    if (!args) {
```

Comment:
Formatting/Indentation seems off in all commands

Reply:
Fixed